infrastructure/src/test/java/org/corfudb/infrastructure/log/statetransfer/StateTransferManagerTest.java

```diff
@@ -460,6 +464,103 @@ void testRangeToBatchRequestStreamWithServers() {
         assertThat(collect).isEqualTo(expected);
     }
 
+    boolean sequential(List<Long> batch) {
+        if (batch.size() < 2) {
+            return true;
+        }
+        for (int i = 0; i < batch.size() - 1; i++) {
+            if (batch.get(i) + 1 != batch.get(i + 1)) {
```

Comment:
use "equals", otherwise if your number is bigger than 127 `!=` won't work properly

Reply:
done